dynomitemanager/src/main/java/com/netflix/dynomitemanager/defaultimpl/InjectedWebListener.java

```diff
@@ -16,48 +16,43 @@
 
 package com.netflix.dynomitemanager.defaultimpl;
 
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.quartz.SchedulerFactory;
+import org.quartz.impl.StdSchedulerFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.google.common.collect.Lists;
 import com.google.inject.AbstractModule;
 import com.google.inject.Guice;
 import com.google.inject.Injector;
 import com.google.inject.Module;
-import com.google.inject.Scopes;
 import com.google.inject.servlet.GuiceServletContextListener;
 import com.google.inject.servlet.ServletModule;
 import com.netflix.dynomitemanager.FloridaServer;
-import com.netflix.dynomitemanager.sidecore.IConfiguration;
-import com.netflix.dynomitemanager.sidecore.ICredential;
-import com.netflix.dynomitemanager.sidecore.aws.IAMCredential;
-import com.netflix.dynomitemanager.sidecore.config.AwsInstanceDataRetriever;
-import com.netflix.dynomitemanager.sidecore.config.InstanceDataRetriever;
-import com.netflix.dynomitemanager.sidecore.config.LocalInstanceDataRetriever;
-import com.netflix.dynomitemanager.sidecore.scheduler.GuiceJobFactory;
-import com.netflix.dynomitemanager.defaultimpl.FloridaStandardTuner;
-import com.netflix.dynomitemanager.sidecore.utils.ProcessTuner;
 import com.netflix.dynomitemanager.IFloridaProcess;
-import com.netflix.dynomitemanager.defaultimpl.FloridaProcessManager;
-import com.netflix.dynomitemanager.identity.AwsInstanceEnvIdentity;
 import com.netflix.dynomitemanager.identity.CassandraInstanceFactory;
+import com.netflix.dynomitemanager.identity.DefaultVpcInstanceEnvIdentity;
 import com.netflix.dynomitemanager.identity.IAppsInstanceFactory;
 import com.netflix.dynomitemanager.identity.InstanceEnvIdentity;
-import com.netflix.dynomitemanager.identity.LocalInstanceEnvIdentity;
-import com.netflix.dynomitemanager.sidecore.storage.RedisStorageProxy;
+import com.netflix.dynomitemanager.sidecore.IConfiguration;
+import com.netflix.dynomitemanager.sidecore.ICredential;
+import com.netflix.dynomitemanager.sidecore.aws.IAMCredential;
+import com.netflix.dynomitemanager.sidecore.config.InstanceDataRetriever;
+import com.netflix.dynomitemanager.sidecore.config.VpcInstanceDataRetriever;
 import com.netflix.dynomitemanager.sidecore.storage.IStorageProxy;
-import com.netflix.dynomitemanager.supplier.EurekaHostsSupplier;
+import com.netflix.dynomitemanager.sidecore.storage.RedisStorageProxy;
+import com.netflix.dynomitemanager.sidecore.utils.FloridaHealthCheckHandler;
+import com.netflix.dynomitemanager.sidecore.utils.ProcessTuner;
 import com.netflix.dynomitemanager.supplier.HostSupplier;
 import com.netflix.dynomitemanager.supplier.LocalHostsSupplier;
+import com.netflix.karyon.spi.HealthCheckHandler;
```

Comment:
We do not currently use the FloridaHealthCheckHandler. The healthcheck is currently handled by a scheduler running every 15 seconds: [ProcessMonitor](https://github.com/Netflix/dynomite-manager/blob/dev/dynomitemanager/src/main/java/com/netflix/dynomitemanager/FloridaServer.java#L139-L140). Doesn't this work for you?